### PR TITLE
Maps with big pills

### LINKED ISSUE #162: More maps with big pills and terrified ghosts
We need more maps that involve #81 and #153 

Since Mr @ForNeVeR is really experienced game level designer, I'm assigning this task to him.

### PR DESCRIPTION
**Pull request by rexim**
Close #162

### CODE REVIEW

**Comment by rexim** [comment]
Blocked by #172

File: maps/map07.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+##########
+#o.     ##
+ ######  
+      ####
+#### ###..
+   # #. #
+##    #.##
+   #  #..#
+...#.....#
+##########
```
Comment (ForNeVeR) [inline maps/map07.txt]: Are you sure _this_ map is useful? Maybe we shouldn't repeat this boring "tutorial" map again?

Comment (rexim) [inline maps/map07.txt]: I think you're right. I'll remove it.

**Comment by ForNeVeR** [comment]
I have no other complaints, feel free to merge after that.

**Pull request merged**